tests/test_modeling_t5.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, parent):
         self.batch_size = 13
         self.encoder_seq_length = 7
         self.decoder_seq_length = 9
+        # For common tests
+        self.seq_length = self.decoder_seq_length
```

Comment:
@patrickvonplaten this value is needed for the labels building. Is it alright if I add this?